idaes/models_extra/gas_solid_contactors/properties/methane_iron_OC_reduction/hetero_reactions.py

```diff
@@ -456,8 +456,9 @@ def rate_constant_eqn(b, j):
                             )
                             * self.solid_state_ref.temperature
                         )
-                    )
+                    ) * 1E-3)
```

Comment:
Manually scaling the constraints isn't needed. For the failing  test, which is "test_initialize_unscaled" (I think), I suggest reducing the time_set to 360s and nfe to 10 in the iron_oc_unscaled method. Ideally, at the given initial conditions, and time horizon of 3600 you will want to apply a scaling transformation before solving this model.